python/foxglove-sdk/python/docs/api/index.rst

```diff
@@ -24,3 +65,44 @@ Channels
    :maxdepth: 1
 
    ./channels
+
+Parameters
+^^^^^^^^^^
+
+Used with the parameter service during live visualization. Requires the :py:data:`Capability.Parameters` capability.
+
+.. autoclass:: foxglove.ParameterType
+
+   .. py:data:: ByteArray
+
+      A byte array, encoded as a base64-encoded string.
+
+   .. py:data:: Float64
+
+      A decimal or integer value that can be represented as a 64-bit floating point number.
+
+   .. py:data:: Float64Array
+
+      An array of decimal or integer values that can be represented as 64-bit floating point numbers.
+
+.. autoclass:: foxglove.ParameterValue
+
+   .. py:class:: Number(value: float)
+
+      A decimal or integer value.
+
+   .. py:class:: Bool(value: bool)
+
+      A boolean value.
+
+   .. py:class:: Bytes(value: bytes)
+
+      A byte array
```

Comment:
Should this match the description In `_foglove_py/__init__.pyi`?

```suggestion
      A byte array, encoded as a base64-encoded string.
```

Reply:
Good catch. It's the other way around; the python implementation does the encoding for you. I've updated.

Reply:
Ah I see, in that case I think this is just missing a period.